lib/rules/web/ocm_console/install_page.xyaml

```diff
@@ -800,4 +800,82 @@ check_creation_page_cloud_tab_common_part:
 check_creation_page_cloud_tab_without_quota:
     action: creation_cards_page_loaded
     action: check_creation_cards_page_title
-    action: check_creation_page_cloud_tab_common_part
\ No newline at end of file
+    action: check_creation_page_cloud_tab_common_part
+switch_to_aws_arm_installer_page:
+    action: switch_to_creation_cards_page
+    action: click_aws_arm_installer_button
+check_download_installer_button:
+    element:
+        selector:
+            xpath: //a[@class='pf-c-button pf-m-secondary download-button tool-arminstaller' and text()='Download installer']
+check_download_pull_secret_button:
+    element:
+        selector:
+            xpath: //button[@class='pf-c-button pf-m-secondary' and text()='Download pull secret']
+check_copy_pull_secret_button:
+    element:
+        selector:
+            xpath: //button[@class='pf-c-button pf-m-link' and text()='Copy pull secret']
+check_download_clt_button:
+    element:
+        selector:
+            xpath: //a[@class='pf-c-button pf-m-secondary download-button tool-cli_tools' and text()='Download command-line tools']
+check_aws_arm_install_page_browser_title:
+    scripts:
+    - command: return document.title=='Install OpenShift 4 | AWS ARM | Experimental Developer Preview Builds'
+      expect_result: true
+check_aws_arm_installer_page_title:
+    element:
+        selector:
+            xpath: //h1[text()='Install OpenShift Container Platform 4']
+check_logo_on_aws_arm_install_page_title:
+    element:
+        selector:
+            xpath: //img[@src='/apps/chrome/js/fonts/logo.svg']
+check_links_on_install_page:
+    element:
+        selector:
+            xpath: //a[@href='<link>']
+check_linux_download_installer_button_in_aws_arm_install_page:
+    params:
+        os_name: Linux
+        os_link: linux-amd64.tar.gz
+        installer_type: aarch64/clients/ocp-dev-preview/pre-release
+    action: check_download_installer_button_in_install_page
```

Comment:
Missed a checkpoint that when the 'Architecture type' is 'aarch64'

Reply:
I don't quite get your point here. When I select "Linux" and "x86_64" as arch type, the installer type for the download button is "aarch64/clients/ocp-dev-preview/pre-release" but not "x86_64/clients/ocp-dev-preview/pre-release"

Reply:
Please check, the install files are different when you choose 'x86_64' and 'aarch64'. I mean you only check 'x86_64', but miss 'aarch64'.